bundles/org.openhab.binding.mikrotik/src/main/java/org/openhab/binding/mikrotik/internal/model/RouterosRouterboardInfo.java

```diff
@@ -51,4 +52,9 @@ public String getFirmwareType() {
     public String getFirmwareVersion() {
         return getProp("current-firmware", "Unknown");
     }
+
+    public boolean hasBuiltinCapsMan() {
+        return ModuleDescriptor.Version.parse(getFirmwareVersion())
+                .compareTo(ModuleDescriptor.Version.parse("7.13")) >= 1;
```

Comment:
You might want to introduce something more generic here. I guess that a range of versions can have this `hasBuiltinCapsMan`
If the version is always numeric, you can parse it to a double and do a greather then or between condition.

Reply:
Thank you for the comment.

> You might want to introduce something more generic here. I guess that a range of versions can have this hasBuiltinCapsMan.

In the [RouterOS 7.13 changelog](https://mikrotik.com/download/changelogs), the following is noted:
> 2. Drivers for older wireless and 60GHz interfaces, as well as the wireless management system CAPsMAN, are now part of a separate "wireless" package instead of being part of the bundle package. 


Regarding version parsing and comparison: I was looking for a solution without introducing new dependencies into the project, but I also wanted to avoid reinventing the wheel. In this particular case, Mikrotik uses major.minor.patch versioning. Therefore, using `Double.parseDouble` here is not possible. If writing a custom implementation for version parsing is necessary, let me know.

Reply:
I cannot fully asses how mandatory this is. But to prevent regressions, it would be nice to have such verison check in place right away.